CLDR-19478 Add how to update email address instructions (#5726)
unicode-org/cldr: docs/site/index/survey-tool/managing-users.md

![image](../../images/index/managingUsers5.jpg)

## To update an email address

> **Make sure you let your users know to use their updated email address as their log in after the update.**

Reviewer: ```suggestion
**Make sure you let your users know to use their updated email address as their login after the update.**
```

1. Open the user that you need to update the email address for by clicking on the magnifying glass icon to the left of the user's name. This will open the user view.

> 2. In the Action column, pick Change E-mail for the user you want to update the email address for.

Reviewer: ```suggestion
2. In the Action column, pick 'Change E-mail...' in the drop-down menu.
```

Reviewer: After step 1, there is only one user available so no need to specify further.

> 3. Add the email address in both the 'New E-mail' and 'New E-mail (confirm)' boxes.

Reviewer: ```suggestion
3. Add the new email address in both the 'New E-mail' and 'New E-mail (confirm)' boxes.
```

> 3. Add the email address in both the 'New E-mail' and 'New E-mail (confirm) boxes 

Reviewer: ```suggestion
3. Add the email address in both the 'New E-mail' and 'New E-mail (confirm)' boxes.
```
